pkg/vcs/testos.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package vcs
+
+import (
+	"io"
+)
+
+type Testos struct {
+	*git
+}
+
+func NewTestos(dir string) *Testos {
+	return &Testos{
+		git: newGit(dir, nil),
+	}
+}
+
+func (ctx *Testos) Bisect(bad, good string, trace io.Writer, pred func() (BisectResult, error)) ([]*Commit, error) {
```

Comment:
I think you can simply delete this method, it will be "inherited" from git anyway.